src/main/java/org/ossgang/commons/observable/Observables.java

```diff
@@ -31,10 +37,77 @@ private Observables() {
      * return the given value on get(). On subscribe(), it will immediately send the given value to the listener.
      *
      * @param value the value
-     * @param <T> the value type
+     * @param <T>   the value type
      * @return a constant ObservableValue
      */
     public static <T> ObservableValue<T> constant(T value) {
         return new ConstantObservableValue<>(value);
     }
+
+    /**
+     * Merges any Observables of the same type into one. Updates whenever all sources have provided at least one item.
+     */
+    @SafeVarargs
+    public static <K, V> ObservableValue<Map<K, V>> zip(Function<K, ObservableValue<V>> supplier, K... items) {
+        return zip(supplier, Arrays.asList(items));
+    }
+
+    /**
+     * Merges any Observables of the same type into one. Updates whenever all sources have provided at least one item.
+     */
+    public static <K, V> ObservableValue<Map<K, V>> zip(Function<K, ObservableValue<V>> supplier, Collection<K> items) {
```

Comment:
It would be nice to be consistent with signatures:
public static <K, V> ObservableValue<Map<K, V>> zip(Map<K,ObservableValue<V>> values);
public static <K, V, O> ObservableValue<O> zip(Map<K,ObservableValue<V>> values, Function<Map<K,V>, O> mapper);
public static <V> ObservableValue<List<V>> zip(List<ObservableValue<V>> values);
public static <V, O> ObservableValue<O> zip(List<ObservableValue<V>> values, Function<List<V>, O> mapper);

(with one common base implementation of course)

Reply:
Done for mergeLatest

Reply:
Done for the zip